### ISSUE TITLE
v2raya: 0.0.0.0 fails to start management interface on 2017

### ISSUE THREAD

**Issue by wulfy23** [openwrt/packages#16511]
Maintainer: @1715173329
Environment: r1745x

Description:

```
lsof -i -nP | grep v2ray | grep -v 443 | wc -l
0
```

as stated in the issue title... possibly modify initscript to select br-lan ip if 2017 mgmt ip is 0.0.0.0

**Comment by cubercsl** [comment]
https://github.com/openwrt/packages/blob/90ec599f9ef7aa011febe5528ba8b0b2016bfe02/net/v2raya/files/v2raya.init#L40

The param_arg `--pluginlistenport`  has been deprecated in 1.5.2. Just commet it out and try again.

**Comment by 1715173329** [comment]
I apologize for this problem, will fix this in next release.
Please see the reply above for solution.

**Comment by wulfy23** [comment]
sweet. thankyou for the work on this package and the fast response.

**Comment by predators46** [comment]
Hy 1715173329

does v2raya not have service v2raya restart command i get error

`root@OpenWrt:/tmp# /etc/init.d/v2raya restart
Command failed: Not found`

**Comment by predators46** [comment]
v2raya is great but I gave up using it. there is no information to access the web gui v2raya I compared with adguardhome very complete information when it finished installing in the system-log. to access the v2raya web gui I have to enter the v2raya command in the terminal and wait for the information to access v2raya example 192.168.1.10:2017 but if the terminal is closed or the router is rebooted I can no longer access the v2raya web gui and I enter the v2raya command in the terminal and get an error message `2021/09/07 03:40:41 nutsdb.go:23: db.buildIndexes error: invalid argument`

**Comment by cubercsl** [comment]
> to access the v2raya web gui I have to enter the v2raya command in the terminal and wait for the information to access v2raya example 192.168.1.10:2017 but if the terminal is closed or the router is rebooted I can no longer access the v2raya web gui and I enter the v2raya command in the terminal and get an error message `2021/09/07 03:40:41 nutsdb.go:23: db.buildIndexes error: invalid argument`

https://github.com/openwrt/packages/issues/16511#issuecomment-909992875

```
uci set v2raya.config.enabled='1'
uci commit v2raya
/etc/init.d/v2raya enable
/etc/init.d/v2raya start
```

**Comment by predators46** [comment]
> > to access the v2raya web gui I have to enter the v2raya command in the terminal and wait for the information to access v2raya example 192.168.1.10:2017 but if the terminal is closed or the router is rebooted I can no longer access the v2raya web gui and I enter the v2raya command in the terminal and get an error message `2021/09/07 03:40:41 nutsdb.go:23: db.buildIndexes error: invalid argument`
> 
> @predators46
> You can set autostart v2raya.
> 
> [#16511 (comment)](https://github.com/openwrt/packages/issues/16511#issuecomment-909992875)
> 
> ```
> uci set v2raya.config.enabled='1'
> uci commit v2raya
> /etc/init.d/v2raya enable
> /etc/init.d/v2raya start
> ```

I've followed everything and it doesn't work when I reboot my router the v2raya web gui can't be accessed anymore

**Comment by 1715173329** [comment]
@predators46 you need modify the init script...
See the second comment.

**Comment by predators46** [comment]
> @predators46 you need modify the init script...
> See the second comment.

I've changed the init script when the router reboots I can't access the v2raya web gui anymore

**Comment by 1715173329** [comment]
> I've changed the init script when the router reboots I can't access the v2raya web gui anymore

Please add the following commands to `start_service()` to see the real errors:
```
procd_set_param stdout 1
procd_set_param stderr 1
```

See https://openwrt.org/docs/guide-developer/procd-init-scripts#defining_service_instances

**Comment by predators46** [comment]
Thanks all solved

**Comment by malikshi** [comment]
i tried to install via opkg install v2rayA, it's seem not available in 21.02.0

**Comment by 1715173329** [comment]
Please try with #16634
